charts/super-agent/values.yaml

```diff
@@ -0,0 +1,181 @@
+helm:
+  # -- Enable the installation of the CRs so FluxCD deploy the Super Agent is deployed. This an advanced/debug flag. It
+  # should be always be true unless you know what you are going. 
+  create: true
```

Comment:
if it is for debugging only, should we expose it?

Reply:
I cannot think of many reasons to disable it but I am thinking of things like the guided install that could leverage it somehow on templating stages.

I prefer to document everything and let the user footgun itself rather than having undocumented things that we would need. At least, at the stage that we are right now, we need all the documentation for ourselves.

Reply:
fair enough